front/src/Administration/TypeMessage.ts

```diff
@@ -0,0 +1,62 @@
+import {TypeMessageInterface} from "./UserMessageManager";
+import {HtmlUtils} from "../WebRtc/HtmlUtils";
+
+export class Ban implements TypeMessageInterface {
+    private nbSecond = 0;
+    private maxNbSecond = 10;
+    private titleMessage = 'IMPORTANT !';
+
+    showMessage(message: string): void {
+        const div : HTMLDivElement = document.createElement('div');
```

Comment:
You should really use .innerHTML to create HTML in Javascript. It is waaaaaaay easier :)